charts/grist/templates/doc_wk_svc.yaml

```
@@ -11,6 +11,7 @@ metadata:
     {{- toYaml $.Values.docWorker.service.annotations | nindent 4 }}
 spec:
   type: {{ .Values.docWorker.service.type }}
+  clusterIP: {{ .Values.docWorker.service.clusterIP }}
```

I think this could be hardcoded, this service was never meant to have an IP
```suggestion
  clusterIP: None
```
and removing the rest of the configuration.

Looking at bitnami charts for example, they do this and add `headless` in the file name.